### ISSUE TITLE
Clean build yields 11 warnings about TraceEvent package

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#18836]
I just did:
```
git clean -xdf
git pull upstream master
build
```
and I get the following warnings as part of the build:
```
C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : Detected package downgrade: Microsoft.Diagnostics.Tracing.TraceEvent from 1.0.3-alpha-experimental to 1.0.0-alpha-experimental  [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj]
C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : XUnit.Runtime (>= 4.4.0) -> xunit.performance.metrics (>= 1.0.0-beta-build0003) -> Microsoft.Diagnostics.Tracing.TraceEvent (>= 1.0.3-alpha-experimental)  [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj]
C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : XUnit.Runtime (>= 4.4.0) -> Microsoft.Diagnostics.Tracing.TraceEvent (>= 1.0.0-alpha-experimental) [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj]
C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : Detected package downgrade: Microsoft.Diagnostics.Tracing.TraceEvent from 1.0.3-alpha-experimental to 1.0.0-alpha-experimental  [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj]
C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : XUnit.Runtime (>= 4.4.0) -> xunit.performance.api (>= 1.0.0-beta-build0003) -> Microsoft.Diagnostics.Tracing.TraceEvent (>= 1.0.3-alpha-experimental)  [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : XUnit.Runtime (>= 4.4.0) -> Microsoft.Diagnostics.Tracing.TraceEvent (>= 1.0.0-alpha-experimental) [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj]
C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : Detected package downgrade: Microsoft.Diagnostics.Tracing.TraceEvent from 1.0.3-alpha-experimental to 1.0.0-alpha-experimental  [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj]
C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : XUnit.Runtime (>= 4.4.0) -> xunit.performance.api (>= 1.0.0-beta-build0003) -> Microsoft.Diagnostics.Tracing.TraceEvent (>= 1.0.3-alpha-experimental)  [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj]
C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : XUnit.Runtime (>= 4.4.0) -> Microsoft.Diagnostics.Tracing.TraceEvent (>= 1.0.0-alpha-experimental) [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj]
C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : Detected package downgrade: Microsoft.Diagnostics.Tracing.TraceEvent from 1.0.3-alpha-experimental to 1.0.0-alpha-experimental  [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj]
C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : XUnit.Runtime (>= 4.4.0) -> xunit.performance.metrics (>= 1.0.0-beta-build0003) -> Microsoft.Diagnostics.Tracing.TraceEvent (>= 1.0.3-alpha-experimental)  [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj]
C:\Users\stoub\Source\repos\corefx\Tools\dotnetcli\sdk\2.0.0-preview1-005724\NuGet.targets(97,5): warning : XUnit.Runtime (>= 4.4.0) -> Microsoft.Diagnostics.Tracing.TraceEvent (>= 1.0.0-alpha-experimental) [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj] [C:\Users\stoub\Source\repos\corefx\external\test-runtime\XUnit.Runtime.depproj]
```
cc: @weshaggard, @ericstj, @brianrob

**Comment by brianrob** [comment]
@drewscoggins, @jorive can you please take a look. This is the test build pulling in multiple versions of traceevent.

**Comment by brianrob** [comment]
@DrewScoggins, does this need to be fixed for 2.0?  If not, let's move this out.

**Comment by weshaggard** [comment]
I just submitted a PR to update the versions. https://github.com/dotnet/corefx/pull/19010
